Rubberduck.Resources/RubberduckUI.de.resx

```diff
@@ -1685,4 +1685,22 @@ Eine private Klasse kann eine private Schnittstelle implementieren.</value>
   <data name="ExtractInterface_InstancingGroupBox" xml:space="preserve">
     <value>Sichtbarkeit</value>
   </data>
+  <data name="ExtractInterface_ImplementationOptionsGroupBox" xml:space="preserve">
+    <value>Implementation Optionen:</value>
+  </data>
+  <data name="ExtractInterface_OptionAddEmptyImplementation" xml:space="preserve">
+    <value>Füge leere Implementation hinzu</value>
+  </data>
+  <data name="ExtractInterface_OptionReplaceMembersWithInterfaceMembers" xml:space="preserve">
+    <value>Ersetze Members durch Interface Members</value>
```

Comment:
"Member" is often translated as "Element" in the newer translations. There also was a PR to adjust that for a variety of strings (#5585)